website/docs/quick-start-guide.md

```diff
@@ -20,6 +20,7 @@ Hudi works with Spark-2.4.3+ & Spark 3.x versions. You can follow instructions [
 
 | Hudi            | Supported Spark 3 version                       |
 |:----------------|:------------------------------------------------|
+| 0.12.x          | 3.3.x (default build), 3.2.x, 3.1.x             |
```

Comment:
Why does `0.11.x` has `Spark bundle only`?

Reply:
In 0.11.x, when both Spark bundle and utilities bundle are used together, there is a compatibility issue.  That has been resolved in 0.12.0.